_includes/talk.html

```diff
@@ -50,6 +50,17 @@
           <span>Watch video</span>
         </a>
       {% endif %}
+      {% if include.audio %}
+        <a href="{{ include.audio}}" class="button is-text">
+          <span class="icon is-small">
+            <img
+              src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/solid/music.svg"
```

Comment:
This icon might be slightly more appropriate, given that it's not music :) I'll apply the change.

```suggestion
              src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/solid/file-audio.svg"
```